src/Tools/Common/Commands/ProcessStatus.cs

```diff
@@ -3,29 +3,67 @@
 // See the LICENSE file in the project root for more information.
 
 using Microsoft.Diagnostics.NETCore.Client;
+using Microsoft.Tools.Common;
 using System;
 using System.CommandLine;
 using System.CommandLine.Invocation;
 using System.CommandLine.IO;
 using System.Diagnostics;
 using System.Linq;
+using System.Runtime.InteropServices;
 using System.Text;
 using Process = System.Diagnostics.Process;
+using System.IO;
+using System.ComponentModel;
+using System.Threading.Tasks;
+using Microsoft.Diagnostics.Tools.Trace.CommandLine;
+using System.CommandLine.Binding;
 
 namespace Microsoft.Internal.Common.Commands
 {
     public class ProcessStatusCommandHandler
     {
-        public static Command ProcessStatusCommand(string description) =>
-            new Command(name: "ps", description)
+        public static Command ProcessStatusCommand() =>
+            new Command(name: "ps", 
+            description: "Lists the dotnet processes that traces can be collected")
             {
-                Handler = CommandHandler.Create<IConsole>(PrintProcessStatus)
+                HandlerDescriptor.FromDelegate((ProcessStatusDelegate)ProcessStatus).GetCommandHandler(),
+                VerboseOption()
             };
 
+        public static Option VerboseOption() =>
+            new Option(
+                aliases: new[] {"-v", "--verbose"},
+                description: "Output includes commandline arguments.")
+                {
+                    Argument = new Argument<bool>(name: "verbose", getDefaultValue: () => false)
+                };
```

Comment:
Is there a specific reason to put this behind a flag? I think this should be on by default.

Reply:
I put it behind a flag because it was slightly slower with the verbose option, but other than that there is no reason to have it behind a flag.

Reply:
Interesting! How much slower are we talking? 1.5x or 10x?

Reply:
I timed it today, and they are about the same speed, I used 11 samples of running with verbose and 11 more without verbose. It turns out that they are about the same speed with or without the verbose flag. I will remove the flag.